requirements/default.txt

```diff
@@ -6,3 +6,4 @@ vispy>=0.5.3
 PyOpenGL>=3.1.0
 PyQt5>=5.10.1
 SIP>=0.1.5
+pre-commit>=1.16.1
```

Comment:
this is a dev-only tool so it doesn't belong in the default requirements

Reply:
okay removing

Reply:
I would say, let's add `requirements/development.txt` which includes pre-commit and black.